x-pack/plugins/security/server/prompt_page.tsx

```diff
@@ -60,16 +89,12 @@ export function PromptPage({
     `${basePath.serverBasePath}/ui/legacy_light_theme.css`,
   ];
 
-  // Emotion SSR styles will be prepended to the <body> and emit a console log warning about :first-child selectors
-  const emotionCache = createCache({
-    key: 'css',
-    prepend: true,
-  });
-
   return (
     <html lang={i18n.getLocale()}>
       <head>
         <title>Elastic</title>
+        {/* eslint-disable-next-line react/no-danger */}
+        <style dangerouslySetInnerHTML={{ __html: `</style>${emotionStyles}` }} />
```

Comment:
A quick explanation on why this creates a blank `<style></style>` tag:

1. Since Emotion is returning HTML as one large string (including specific classes/attributes on its `<style>` tags that we likely should not ignore), we need to use `dangerouslySetInnerHTML`
2. The number of children [allowed within the `<head>` tag is limited](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head#see_also), e.g. no `<div>`s or otherwise handy 'grouping' type tag. If you try to use a tag that normally does not have children, e.g. the `<meta>` tag, Kibana/React will error with ```[ERROR][http.server.Kibana] Error: meta is a void element tag and must neither have `children` nor use `dangerouslySetInnerHTML`.```
3. `style` was therefore the tag that made the most sense, but causes weird/unnecessary nesting behavior, hence the prepended `<style>`. [See this StackOverflow example for how this works.](https://stackoverflow.com/a/60938902/4294462). This ends up creating a blank `<style></style>` tag, but that really isn't the worst thing in the world, and is an unfortunate necssary evil until React supports `dangerouslySetInnerHTML` on fragments.